cmd/operator-sdk/new/cmd.go

```diff
@@ -99,8 +95,9 @@ generates a default directory layout based on the input <project-name>.
 `,
 		RunE: newFunc,
 	}
-	newCmd.Flags().StringVar(&operatorType, "type", "go",
-		"Type of operator to initialize (choices: \"go\", \"ansible\" or \"helm\")")
+
+	newCmd.Flags().StringVar(&operatorType, "type", "ansible",
```

Comment:
Currently, to scaffold new ansible or helm projects, users have to specify `--type`. Should we make `--type` a required flag so the UX stays the same?

It seems strange to prefer one operator type over another between ansible and helm.

```suggestion
	newCmd.Flags().StringVar(&operatorType, "type", "",
```

Reply:
Done